frontend/src/v5/ui/controls/assigneesSelect/assigneesSelectMenu/assigneesSelectMenu.component.tsx

```diff
@@ -44,88 +42,76 @@ export const AssigneesSelectMenu = ({
+	}, [filteredItems.length]);
 
 	return (
-		<SearchContextComponent filteringFunction={filterItems} items={[...jobs, ...users]}>
-			<SearchContext.Consumer>
-				{ ({ filteredItems }: SearchContextType<typeof MenuItem>) => {
-					const [filteredUsers, filteredJobs] = partition(filteredItems, isUser);
-					return (
-						<HiddenSelect
-							open={open}
-							value={value || []}
-							multiple
-							MenuProps={{
-								disableAutoFocusItem: true,
-								PaperProps: {
-									style: { maxHeight: 531, maxWidth: 218 },
-								},
-							}}
-							onClick={onClickList}
-							{...props}
-						>
-							<SearchInputContainer>
-								<SearchInput
-									placeholder={formatMessage({ id: 'searchSelect.searchInput.placeholder', defaultMessage: 'Search...' })}
-									onClick={preventPropagation}
-									onKeyDown={preventPropagation}
-								/>
-							</SearchInputContainer>
-							<ListHeading>
-								<FormattedMessage id="assigneesSelectMenu.jobsHeading" defaultMessage="Jobs" />
-							</ListHeading>
-							{filteredJobs.length > 0 && filteredJobs.map((job) => (
-								<AssigneesSelectMenuItem
-									key={job._id}
-									value={job._id}
-									icon={() => <JobPopoverCircle job={job} />}
-									title={job._id}
-								/>
-							))}
-							{!filteredJobs.length && (
-								<NoResults>
-									<FormattedMessage
-										id="form.searchSelect.menuContent.emptyList"
-										defaultMessage="No results"
-									/>
-								</NoResults>
-							)}
-							<HorizontalRule />
-							<ListHeading>
-								<FormattedMessage id="assigneesSelectMenu.usersHeading" defaultMessage="Users" />
-							</ListHeading>
-							{filteredUsers.length > 0 && filteredUsers.map(({ user, firstName, lastName, job }) => (
-								<AssigneesSelectMenuItem
-									key={user}
-									value={user}
-									icon={() => <UserPopoverCircle username={user} />}
-									title={`${firstName} ${lastName}`}
-									subtitle={job}
-								/>
-							))}
-							{!filteredUsers.length && (
-								<NoResults>
-									<FormattedMessage
-										id="form.searchSelect.menuContent.emptyList"
-										defaultMessage="No results"
-									/>
-								</NoResults>
-							)}
-						</HiddenSelect>
-					);
-				}}
-			</SearchContext.Consumer>
-		</SearchContextComponent>
+		// @ts-ignore
+		<HiddenSelect
+			open={open}
+			value={value || []}
+			onClick={onClickList}
+			{...props}
+		>
+			<SearchInputContainer>
+				<SearchInput
+					placeholder={formatMessage({ id: 'searchSelect.searchInput.placeholder', defaultMessage: 'Search...' })}
+					onClick={preventPropagation}
+					onKeyDown={preventPropagation}
+				/>
+			</SearchInputContainer>
+			<ListSubheader>
+				<FormattedMessage id="assigneesSelectMenu.jobsHeading" defaultMessage="Jobs" />
+			</ListSubheader>
+			{jobs.length > 0 && jobs.map(({ _id }) => (
+				<AssigneesSelectMenuItem
+					key={_id}
+					assignee={_id}
+					value={_id}
```

Comment:
Where `AssigneesSelectMenuItem` is used, `assignee` and `value` are the same (i.e. `_id` here and `user.user`) later. Is there a reason why they are two separate props?